### ISSUE TITLE
Execution failed for task ':react-native-vitals:verifyReleaseResources'

### ISSUE THREAD

**Issue by cristianoccazinsp** [robinpowered/react-native-vitals#12]
While building a release version, I'm getting the following, any ideas?

```
* What went wrong:
Execution failed for task ':react-native-vitals:verifyReleaseResources'.
> java.util.concurrent.ExecutionException: com.android.builder.internal.aapt.v2.Aapt2Exception: Android resource linking failed
  error: resource android:style/TextAppearance.Material.Widget.Button.Borderless.Colored not found.
  error: resource android:style/TextAppearance.Material.Widget.Button.Colored not found.
  /Users/cristiano/Desktop/trabajo/zinspector/mobileRN/zinspector_rn/node_modules/react-native-vitals/android/build/intermediates/res/merged/release/values-v26/values-v26.xml:7: error: resource android:attr/colorError not found.
  /Users/cristiano/Desktop/trabajo/zinspector/mobileRN/zinspector_rn/node_modules/react-native-vitals/android/build/intermediates/res/merged/release/values-v26/values-v26.xml:11: error: resource android:attr/colorError not found.
  /Users/cristiano/Desktop/trabajo/zinspector/mobileRN/zinspector_rn/node_modules/react-native-vitals/android/build/intermediates/res/merged/release/values-v26/values-v26.xml:15: error: style attribute 'android:attr/keyboardNavigationCluster' not found.
  /Users/cristiano/Desktop/trabajo/zinspector/mobileRN/zinspector_rn/node_modules/react-native-vitals/android/build/intermediates/res/merged/release/values-v28/values-v28.xml:7: error: resource android:attr/dialogCornerRadius not found.
  /Users/cristiano/Desktop/trabajo/zinspector/mobileRN/zinspector_rn/node_modules/react-native-vitals/android/build/intermediates/res/merged/release/values-v28/values-v28.xml:11: error: resource android:attr/dialogCornerRadius not found.
  /Users/cristiano/Desktop/trabajo/zinspector/mobileRN/zinspector_rn/node_modules/react-native-vitals/android/build/intermediates/res/merged/release/values/values.xml:2711: error: resource android:attr/fontStyle not found.
  /Users/cristiano/Desktop/trabajo/zinspector/mobileRN/zinspector_rn/node_modules/react-native-vitals/android/build/intermediates/res/merged/release/values/values.xml:2712: error: resource android:attr/font not found.
  /Users/cristiano/Desktop/trabajo/zinspector/mobileRN/zinspector_rn/node_modules/react-native-vitals/android/build/intermediates/res/merged/release/values/values.xml:2713: error: resource android:attr/fontWeight not found.
  /Users/cristiano/Desktop/trabajo/zinspector/mobileRN/zinspector_rn/node_modules/react-native-vitals/android/build/intermediates/res/merged/release/values/values.xml:2714: error: resource android:attr/fontVariationSettings not found.
  /Users/cristiano/Desktop/trabajo/zinspector/mobileRN/zinspector_rn/node_modules/react-native-vitals/android/build/intermediates/res/merged/release/values/values.xml:2715: error: resource android:attr/ttcIndex not found.
  /Users/cristiano/Desktop/trabajo/zinspector/mobileRN/zinspector_rn/node_modules/react-native-vitals/android/build/intermediates/res/merged/release/values/values.xml:2796: error: resource android:attr/startX not found.
  /Users/cristiano/Desktop/trabajo/zinspector/mobileRN/zinspector_rn/node_modules/react-native-vitals/android/build/intermediates/res/merged/release/values/values.xml:2799: error: resource android:attr/startY not found.
  /Users/cristiano/Desktop/trabajo/zinspector/mobileRN/zinspector_rn/node_modules/react-native-vitals/android/build/intermediates/res/merged/release/values/values.xml:2802: error: resource android:attr/endX not found.
  /Users/cristiano/Desktop/trabajo/zinspector/mobileRN/zinspector_rn/node_modules/react-native-vitals/android/build/intermediates/res/merged/release/values/values.xml:2805: error: resource android:attr/endY not found.
  /Users/cristiano/Desktop/trabajo/zinspector/mobileRN/zinspector_rn/node_modules/react-native-vitals/android/build/intermediates/res/merged/release/values/values.xml:2813: error: resource android:attr/offset not found.
  error: failed linking references.
```

**Comment by cristianoccazinsp** [comment]
Fixed updating gradle build options...

```
android {
    compileSdkVersion rootProject.ext.compileSdkVersion
    buildToolsVersion rootProject.ext.buildToolsVersion

    defaultConfig {
        minSdkVersion rootProject.ext.minSdkVersion
        targetSdkVersion rootProject.ext.targetSdkVersion
        versionCode 1
        versionName "1.0"
    }
    lintOptions {
        abortOnError false
    }
}
```

https://github.com/robinpowered/react-native-vitals/pull/13

**Comment by christophermark** [comment]
Resolved in #13
